src/gui/symbology-ng/qgssymbollayerv2widget.cpp

```diff
@@ -1588,58 +1617,58 @@ class QgsSvgListModel : public QAbstractListModel
     {
       Q_UNUSED( parent );
       return mSvgFiles.count();
-    }
+  }
```

Comment:
It's a bug in prepare-commit.sh/astyle, if I try to fix things, prepare-commit.sh will reproduce the same strange indentation.

Fixing it manually is a trap for the next one touching the file and using prepare-commit.sh... but I don't see any other option. What do you think ?